From e35bac672a23faec5470d08579db5e5eeba78bce Mon Sep 17 00:00:00 2001
From: Augsch123 <130238014+Augsch123@users.noreply.github.com>
Date: Wed, 17 May 2023 09:57:33 +0800
Subject: [PATCH 1/8] Increase amdgpu_psp timeout

The timeout of some psp commands seems to be too strict for resuming amdgpu in
dom0, resulting in occasional resume failures on some AMD hardwares.
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_psp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_psp.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_psp.c
index c14f63cefe67..1d186dd2ecfd 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_psp.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_psp.c
@@ -264,7 +264,7 @@ static int psp_early_init(struct amdgpu_ip_block *ip_block)
 
 	psp->adev = adev;
 
-	adev->psp_timeout = 20000;
+	adev->psp_timeout = 50000;
 
 	psp_check_pmfw_centralized_cstate_management(psp);
 
-- 
2.50.1

